performance/plugins/extrinsic3rdPartySchedule.ts

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable no-console */
+import { BN } from "@polkadot/util";
+import { TestParams } from "../testParams";
+import { KeyringPair } from "@polkadot/keyring/types";
+import { runTransactions } from "./testRunner";
+import { performanceTestItem } from "./performanceTestItem";
+import { Assets } from "../../utils/Assets";
+import { getApi, initApi } from "../../utils/api";
+import { Node } from "../../utils/Framework/Node/Node";
+import { UserFactory, Users } from "../../utils/Framework/User/UserFactory";
+import { Keyring } from "@polkadot/api";
+import { MGA_ASSET_ID } from "../../utils/Constants";
+import { Sudo } from "../../utils/sudo";
+import { Xyk } from "../../utils/xyk";
+import { ProofOfStake } from "../../utils/ProofOfStake";
+import { setupApi, setupUsers } from "../../utils/setup";
+import { MangataInstance } from "@mangata-finance/sdk";
+
+let tokens: number[] = [];
+export class Extrinsic3rdPartySchedule extends performanceTestItem {
+  constructor() {
+    super();
```

Comment:
Should we rename this variable? Because I think the meaning of the variable is not clear from its name

Reply:
Which one? Tokens?

Reply:
super()